doc/tor.md

```diff
@@ -211,3 +209,96 @@ for normal IPv4/IPv6 communication, use:
   Otherwise it is trivial to link them, which may reduce privacy. Onion
   services created automatically (as in section 2) always have only one port
   open.
+
+- Some states have the ability to monitor and block use of the Tor network.
```

Comment:
nit: Perhaps instead of "Some states have the ability to monitor and block use of the Tor network."

In some regions of the world usage of the Tor network is monitored and/or blocked.